src/main/java/nl/tudelft/ti2206/group9/gui/SettingsScreen.java

```diff
@@ -21,6 +22,8 @@
 
 	/** Hide public constructor. */
 	private SettingsScreen() { }
+    /** Boolean for sound status. */
+    public static boolean sound = true;
```

Comment:
This boolean should be togglable with a toggle method, for example:

    public static final boolean isSoundEnabled() {
        return sound;
    }
    public static final void toggleSound() {
        sound = !sound;
    }
    public static final void toggleSound(boolean newState) {
        sound = newState;
    }